src/actions/AuthActions.js

```diff
@@ -48,24 +47,14 @@ export const logUserIntoFacebook = ({navigate}) => {
     return async (dispatch) => {
         navigate(navKeys.AUTH);
 
-        let token = await AsyncStorage.getItem(tokens.USER_TOKEN);
-
-        if (token) {
-            checkAndSetToken({token});
-        } else {
-            attemptFacebookLogin({dispatch, navigate});
-        }
```

Comment:
This code all gets a lot cleaner now that we can be certain that if a token exists, `redux-persist` would have added it to state already as part of the rehydration process.